polishNotation.py

```diff
@@ -0,0 +1,39 @@
+def polishNotation(expression):
+    s = stack()
+    result = 0
+    for exp in range(0, len(expression)-1):

```

Comment:
|exp| is a poorly named variable. It's an index so it should be named |i| by usual convention.

Also bug: this won't look at the last element. See https://docs.python.org/3/library/stdtypes.html#typesseq-range